### PR TITLE
chore(codeowners): add codeowners for boardsv2

### PR DESCRIPTION
**Pull request by ilgooz**

### CODE REVIEW

**Comment by thehowl** [comment]
note, we're going to do https://github.com/gnolang/gno/pull/2788 once we have a bot that can actually put review requirements before merging, and move towards using codeowners mostly as a way to ping people for reviews.

we can go ahead and merge this for now, just keep in mind that if you want it as a way to be required for review it's eventually going to be a different mechanism.

File: .github/CODEOWNERS
Comment (thehowl) [inline .github/CODEOWNERS:29]: also shouldn't this be /boards/v2?

jeronimo needs to be added with write access to the repo if he is to be a codeowner.

Comment (ilgooz) [reply .github/CODEOWNERS:29]: @moul can you give jeronimoalbi a write access?

@thehowl the name could be changed, we're not actually making a v2 of boards. Potentially boardsv2 will be named just boards. We can figure this out later.

Comment (moul) [reply .github/CODEOWNERS:29]:
```suggestion
/examples/gno.land/p/demo/boardsv2/   @ilgooz @jeronimoalbi @moul
```

Let's add both of you as code owners of this directory. I will also add myself since I am ultimately responsible for this package on behalf of the core team.

File: .github/CODEOWNERS
Comment (moul) [inline .github/CODEOWNERS:40]:
```suggestion
/examples/gno.land/r/demo/boardsv2/   @ilgooz @jeronimoalbi @moul
```

**Review by thehowl — APPROVED** [review]

**Pull request merged**
